src/lucky/tags/input_helpers.cr

```diff
@@ -107,7 +107,7 @@ module Lucky::InputHelpers
   # radio(attribute, "checked_value", attrs: [:required])
   # # => <input type="radio" id="param_key_attribute_name_checked_value" name="param_key:attribute_name" value="checked_value" checked="true" required />
   # ```
-  def radio(field : Avram::PermittedAttribute(String?),
+  def radio(field : Avram::PermittedAttribute(String),
```

Comment:
I'm not entirely sure what these changes entail. How did a non-nilable string attribute behave before?

Reply:
None of my apps really utilize the form stuff much, so I'm not too sure 😕